ipfs-client/.env

```diff
@@ -1,3 +1,3 @@
 MQ_ADDRESS="localhost"
 SERVICE_CHANNEL="ipfs-client"
-NFT_API_KEY="..."
\ No newline at end of file
+NFT_API_KEY="eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJkaWQ6ZXRocjoweGVhNzVBQzEwMmM2QTlCQjc4NDI5NDNlMmMzMUNEMzBmRUNmNUVmMTIiLCJpc3MiOiJuZnQtc3RvcmFnZSIsImlhdCI6MTY0MjQyODUxMDUzMywibmFtZSI6IklQRlMifQ.BjD1EJM1OBWmYClDbRoR1O9vrU3_5-Isb292w3PSSAI"
```

Comment:
This is a free public API - no worries about this being here, more helpful for demo purposes